docs/dev/dev_setup.rst

```diff
@@ -245,7 +253,10 @@ stuff should go. We call this a "PUDL workspace".
 
 First, create a directory to store local caches of raw PUDL data. You can put this
 anywhere, but we put this in ``~/pudl_input`` in the documentation.  Then create an
-environment variable called ``PUDL_INPUT`` to store the path to this new directory:
+environment variable called ``PUDL_INPUT`` to store the path to this new directory and
+make sure that it is set whenever you start up a new shell. These shorthand commands
+will append a line to the end of your shell initialization file. If you need to change
+it later you'll want to edit those files directly:
```

Comment:
Non-blocking: while we're here, we can tell them 

```
Please replace ``/absolute/path/to/pudl_input`` with the actual path to the directory.
```

I've answered several environment questions from people who've copy-pasted without noticing that that's a fake path.